services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -7254,13 +7254,14 @@ public Optional<PushStatusStoreRecordDeleter> getPushStatusStoreRecordDeleter()
   }
 
   /**
-   * @see Admin#getEmergencySourceRegion()
+   * @see Admin#getEmergencySourceRegion(String)
    */
   @Override
-  public Optional<String> getEmergencySourceRegion() {
-    return multiClusterConfigs.getEmergencySourceRegion().equals("")
+  public Optional<String> getEmergencySourceRegion(String clusterName) {
+
+    return multiClusterConfigs.getEmergencySourceRegion(clusterName).equals("")
         ? Optional.empty()
-        : Optional.of(multiClusterConfigs.getEmergencySourceRegion());
+        : Optional.of(multiClusterConfigs.getEmergencySourceRegion(clusterName));
```

Comment:
Nit:
Can we call this function: `multiClusterConfigs.getEmergencySourceRegion(clusterName)` just once?